src/app/screens/ConfirmPayment/index.test.tsx

```diff
@@ -39,4 +38,17 @@ describe("ConfirmPayment", () => {
       await screen.findByText("Remember and set a budget")
     ).toBeInTheDocument();
   });
+
+  test("toggles set budget, displays input with a default budget", async () => {
+    render(
+      <MemoryRouter>
+        <ConfirmPayment {...props} />
+      </MemoryRouter>
+    );
+    fireEvent.click(screen.getByText("Remember and set a budget"));
+    await waitFor(() => screen.getByLabelText("Budget"));
```

Comment:
Would [findby](https://testing-library.com/docs/dom-testing-library/api-async/#findby-queries) work here instead of waitFor+getBy?